include/nudb/detail/format.hpp

```diff
@@ -46,6 +46,21 @@ value size          up to 32 bits (or 32-bit builds can't read it)
 
 static std::size_t constexpr currentVersion = 2;
 
+const std::string& default_dat_file() {
+  static const std::string ddf("nudb.dat");
+  return ddf;
+}
+
+const std::string& default_key_file() {
+  static const std::string dkf("nudb.key");
+  return dkf;
+}
+
+const std::string& default_log_file() {
+  static const std::string dkf("nudb.log");
+  return dkf;
+}
```

Comment:
nudb should have a `<nudb/string_type.hpp>` which declares `nudb::string_view`, copy from here:
https://github.com/boostorg/beast/blob/develop/include/boost/beast/core/string_type.hpp

And these functions should use that `string_view`. Also I think these functions need to be declared `inline`.

Reply:
@miguelportilla will work up a branch that has just this change for us